server/monitor.go

```diff
@@ -1135,6 +1144,18 @@ func (s *Server) createVarz(pcpu float64, rss int64) *Varz {
 		}
 		varz.LeafNode.Remotes = rlna
 	}
+
+	if s.js != nil {
+		s.js.mu.Lock()
+		varz.JetStream = JetStreamVarz{
+			MaxMemory: s.js.config.MaxMemory,
+			MaxStore:  s.js.config.MaxStore,
+			StoreDir:  s.js.config.StoreDir,
+			Accounts:  len(s.js.accounts),
```

Comment:
I am assuming that "Accounts" varies during the lifetime of the server, right? If so, this has to be set in updateVarzRuntimeFields() instead.
The function `createVarz()` is invoked when user call Varz() programmatically, but only once if through the monitoring page. As you can see down below, `createVarz()` then invokes `updateVarzConfigReloadableFields()` and `updateVarzRuntimeFields()`.
So technically, you could set in `createVarz()` the immutable things, but things that can change in config reload or at runtime need to be set in the aforementioned functions.

Reply:
Right, I saw that method but I think I was to jet lagged to really click :) fixed and rebased, keeping an in eye on the tests